setup.cfg

```
@@ -21,16 +21,16 @@ python_requires = >=3.7
 install_requires =
     indexed_gzip >= 0.8.8
     nibabel ~= 3.0
-    nipype ~= 1.4.0
+    nipype ~= 1.4
     nitime
-    niworkflows @ git+https://github.com/nipreps/niworkflows.git@76f831f1f8fbf6c277fdf92535639bbf11d84527
+    niworkflows ~= 1.1.9
     numpy
     pandas
     psutil >= 5.4
     pybids >= 0.10.0
     pyyaml
     sdcflows ~= 1.2.0
-    smriprep ~= 0.5.2
+    smriprep >= 0.6.0rc1
```

```suggestion
    smriprep ~= 0.6.0rc1
```

We should keep this strict to avoid future API changes (i.e. 0.7.x)

---

I might be wrong, but I think `~=` here will not match 0.6.0 (without the rc). What about:
```suggestion
    smriprep >= 0.6.0rc1,<0.7
```

---

yup, that works.